src/config/config.js

```diff
@@ -219,7 +220,10 @@ function setFullDefaults(app) {
     if (e.code && e.code === 'ENOENT') {
       debug(`No defaults found at ${defaultsPath.toString()}`)
     } else {
-      console.log(e)
+      invalidDefaultsFile = true
```

Comment:
we are process.exiting, why the variable?

Reply:
Duh. Cause I dod that before I decided to exit. fixing...